{"uuid":"c0ab4e54-76bb-42ed-ba1c-aa70491ad2d1","historyId":"efcc2ca33b598dbf62ad8b92a5867a99","fullName":"LoginTests.loginWithInvalidCredentials","labels":[{"name":"package","value":"LoginTests"},{"name":"testClass","value":"LoginTests"},{"name":"testMethod","value":"loginWithInvalidCredentials"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Pokupon_project"},{"name":"subSuite","value":"LoginTests"},{"name":"host","value":"MacBook-Air-Alina.local"},{"name":"thread","value":"67529@MacBook-Air-Alina.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"loginWithInvalidCredentials","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Expected error message is not displayed for invalid credentials expected [true] but found [false]","trace":"java.lang.AssertionError: Expected error message is not displayed for invalid credentials expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\n\tat org.testng.Assert.assertTrue(Assert.java:57)\n\tat LoginTests.loginWithInvalidCredentials(LoginTests.java:35)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:819)\n\tat org.testng.TestRunner.run(TestRunner.java:619)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1228)\n\tat org.testng.TestNG.runSuites(TestNG.java:1134)\n\tat org.testng.TestNG.run(TestNG.java:1101)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1705868850163,"stop":1705869042372}